boards/DIM/Src/App/states/App_DriveState.c

```diff
@@ -3,95 +3,6 @@
 
 #define SSEG_HB_NOT_RECEIVED_ERR (888U)
 
-static bool App_DriveState_HasBMSFault()
-{
-    return App_CanRx_BMS_Vitals_CurrentState_Get() == BMS_FAULT_STATE;
-}
-
-static bool App_DriveState_HasDCMFault()
-{
-    // No DCM Faults yet
-    return App_CanRx_DCM_Warnings_MissingHeartbeat_Get();
-}
-
-static bool App_DriveState_HasDIMFault()
-{
-    // No DIM Faults yet
-    return false;
-}
-
-static bool App_DriveState_HasFSMFault()
-{
-    // TODO: Promote this message to fault
-    return App_CanRx_FSM_Warnings_MissingHeartbeat_Get();
-}
```

Comment:
Ooh looks like your git got a little silly

Reply:
You should try to revert all the files which you didn't intentionally edit
You can checkout single files from another branch: https://stackoverflow.com/questions/2364147/how-to-get-just-one-file-from-another-branch
If you do that and commit and push again it shouldn't diff anything here